routes/routes.js

```diff
@@ -0,0 +1,32 @@
+const express = require("express");
+const auth = require("./auth");
+const boards = require("./boards");
+const lists = require("./lists");
+const tasks = require("./tasks");
+const router = express.Router();
+const authMiddleware = require("./auth/authMiddleware");
+const { refreshToken } = require("./auth/token");
+
+router.post("/auth/signUp", async (req, res) => auth.signUp(req, res));
```

Comment:
Please remove unneeded arrow functions
```suggestion
router.post("/auth/signUp", auth.signUp);
```